packages/amplication-client/src/Purchase/FAQ.tsx

```diff
@@ -19,115 +19,118 @@ export const FAQ = () => {
-
-          We will notify you via email or other means when we think it might be a good time for you to consider upgrading your workspace plan.`}
-        />
-        <Question
-          question="What if I need to add more team members?"
-          answer="No problem. Select the plan that meets your needs. The Free edition supports up to two user seats, the Pro edition supports up to five user seats, and the Enterprise edition has no limit. "
+          question="We have legacy systems, and would like to progress to modern architecture. Can you assist with it?"
+          answer={
+            <div>
+              Amplication can faciliate the modernization of your legacy system.
+              We offer a suite of tools and features designed to streamline this
+              process. Those capabilities include:
+              <br />
+              <ul>
+                <li>
+                  <strong>DB Schema Import</strong>: Simplify the migration of
+                  extensive legacy data into new systems, preserving the
+                  integrity and structure of historical data.
+                </li>
+                <li>
+                  <strong>Data Modeling & Event Brokering</strong>: Enables the
+                  transition from monolithic architectures to microservices,
+                  allowing for the creation of scalable and manageable backends.
+                </li>
+                <li>
+                  <strong>Customizable Code Generation</strong>: Ensures
+                  flexibility in development, allowing for specific
+                  customizations without being bound to rigid system
+                  constraints.
+                </li>
+              </ul>
+            </div>
+          }
         />
         <Question
-          question="Can I pay for my subscription with an invoice?"
+          question="Can you assist with the actual deployment (terraform-like) in a cloud provider?"
           answer={
             <div>
-              Companies can use invoices to purchase Enterprise subscriptions.
-              Please{" "}
-              <MailTo
-                email="mailto:sales@amplication.com"
-                subject="Amplication Enterprise Plan Inquiry"
-                body="contact sales"
-              />{" "}
-              for more information and to purchase.
+              Amplication allows you to deploy to AWS's Elastic Container
+              Service (ECS). We provide in-depth documentation that will walk
+              you through using various Amplication plugins to provision the
+              required infrastructure and set up a process for continuous
+              integration and deployment.
+              <br />
+              <br />
+              The steps cover manual configurations for permissions and
```

Comment:
Not sure about this sentence (due to the length of the answer). Again- I think we can link to the documentation page